bin/benchmark.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+java -cp "$(cd `dirname $0`/..; pwd)"'/tools/benchmark-cli/build/libs/benchmark-cli.jar:*' \
```

Comment:
@jakelandis we're not setting an entry point signature on that jar at the moment.
So I just went with the format we use elsewhere (e.g. ingest converter tool) as that seems to work for everyone compatibility wise :)